### PR TITLE
add dummy values to help users understand the cfg format

### PR DESCRIPTION
**Pull request by acxz**
Having some defaults (even if they are dummy values) can help users fill in the corresponding fields.

### CODE REVIEW

File: app/cfg/config.sample.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-    "SQLALCHEMY_DATABASE_URI": "",
+    "SQLALCHEMY_DATABASE_URI": "mariadb+mariadbconnector://user:password@localhost:5000/mysql",
```
Comment (sseshan7) [inline app/cfg/config.sample.json]: Nitpick, can you change `mysql` to `table_name` or something, to better signify that that's the table name, and not some other mysql identifier or something.

File: setup.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
         "pymysql",
         "requests",
         "xlrd",
+        "cryptography",
```
Comment (sseshan7) [inline setup.py]: I added this in a commit yesterday and made it alphabetical :+1:, so I would remove that commit from this PR.

**Review by sseshan7 — CHANGES_REQUESTED** [review]
Tiny changes

**Comment by sseshan7** [comment]
Also, can you rebase main onto this branch

File: app/cfg/config.sample.json (hunk 1 above)
Comment (acxz) [reply app/cfg/config.sample.json]: nice suggestion!

Comment (acxz) [reply app/cfg/config.sample.json]:
```suggestion
    "SQLALCHEMY_DATABASE_URI": "mariadb+mariadbconnector://user:password@localhost:5000/table_name",
```

File: setup.py (hunk 2 above)
Comment (acxz) [reply setup.py]: sweet

**Comment by acxz** [comment]
Done, thx for review

**Review by sseshan7 — APPROVED** [review]
🚢

**Pull request merged**